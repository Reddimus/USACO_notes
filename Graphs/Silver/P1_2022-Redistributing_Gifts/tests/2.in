8
4 2 1 7 3 6 5 8
7 2 5 3 6 1 8 4
6 7 1 3 5 2 8 4
6 2 7 4 1 3 8 5
1 8 2 7 4 5 6 3
8 7 6 1 3 4 2 5
2 7 3 8 4 6 5 1
1 8 4 7 2 3 5 6
