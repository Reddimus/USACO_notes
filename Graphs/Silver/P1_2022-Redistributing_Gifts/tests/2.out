4
7
3
6
5
8
2
1
